pkg/operator/sync.go

```diff
@@ -6,6 +6,8 @@ import (
 	operatorv1 "github.com/openshift/api/operator/v1"
 )
 
+var cleanupDeprecatedDone bool
```

Comment:
nit: I know we set workers=1 when creating controllers. But technically this bool needs to be protected.

Reply:
Done